teuthology/provision/openstack.py

```diff
@@ -127,7 +132,7 @@ def create(self, num, os_type, os_version, arch, resources_hint):
                " --wait " +
                " " + self.basename)
         try:
-            misc.sh(cmd)
+            self.run(cmd, type='compute')
```

Comment:
without this change the **cmd** will use password based authentication instead of reusing an existing token